### ISSUE TITLE
Feature Request: Reliability diagram based on S-Scale

### ISSUE THREAD

**Issue by ktindiana** [ktindiana/sphinxval#52]
SWPC has test UMASEP by checking "if the model predicts and S3, do you measure an S3?" Implement this in SPHINX.

**Comment by rickyegeland** [comment]
Sounds like a generalized MxN contingency table.  People have worked out metrics that deal with these.

**Comment by ktindiana** [comment]
Yes, there are definitely prescribed ways to do the analysis and we can implement something from the literature. I haven't yet identified a source to derive the approach though.

We will also need this because REleASE will issue a three-tiered Quiet, Warning, and Alert state. I think we can validate according to the same analysis, although we will have to put some thought into exactly which observed conditions should be associated with the different states.
